src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,47 @@
+package bit.javaoop;
+
+public class Manager extends Employee {
+
+    private final int N = 5;
+    private Employee tab[] = new Employee[N];
+    private int counter = 0;
+
+    public Manager(String name, double salary,int counter){
+        super(name,salary);
+        this.counter = counter;
+    }
+
+    public int getN() {
+        return N;
+    }
+
+    public Employee[] getTab() {
+        return tab;
+    }
+
+    public void setTab(Employee[] tab) {
+        this.tab = tab;
+    }
+
+    public int getCounter() {
+        return counter;
+    }
+
+    public void setCounter(int counter) {
+        this.counter = counter;
+    }
+
+    public void incrementCounter(){
```

Comment:
Jak robimy takie wewnętrzne pomocnicze metodki to na dwie rzeczy warto zwrócić uwagę:
- Metoda pomocnicza powinna być zawsze w kodzie pod metodą, która jej używa.
- Metody pomocniczne powinny mieć modyfikator przeważnie private w tym wypadku chcemy wykorzysta tą metodkę w klasie dziedziczącej więc private na to nam nie pozwoli ale protected już tak i taki modyfikator powinien się tutaj znaleźć.